crates/rnote-engine/src/store/render_comp.rs

```diff
@@ -6,9 +6,12 @@ use crate::strokes::content::GeneratedContentImages;
 use crate::{Drawable, render};
 use p2d::bounding_volume::{Aabb, BoundingVolume};
 use rnote_compose::ext::AabbExt;
-use rnote_compose::shapes::Shapeable;
+use std::collections::HashMap;
 use tracing::error;
 
+#[cfg(feature = "ui")]
+use rnote_compose::shapes::Shapeable;
```

Comment:
Import dependencies in the guards themselves like it is done in other parts of the code (e.g. L606)